src/ducks/account/helpers.js

```diff
@@ -123,7 +127,15 @@ export const getAccountUpdatedAt = account => {
 
 export const buildHealthReimbursementsVirtualAccount = transactions => {
   const healthExpenses = transactions.filter(transaction => {
-    return isHealthExpense(transaction) && isThisYear(getDate(transaction))
+    if (flag('reimbursement-tag')) {
+      return (
+        isHealthExpense(transaction) &&
+        isThisYear(getDate(transaction)) &&
+        getReimbursementStatus(transaction) === 'pending'
+      )
+    } else {
+      return isHealthExpense(transaction) && isThisYear(getDate(transaction))
+    }
```

Comment:
Nit: I think it would be clearer with a more functional/compositional logic. To show that the flag is only adding one more function and not a different logic.

```js
const filter = _.overEvery([
  isHealthExpense,
  compose(isThisYear, getDate),
  flag('reimbursement-tag') ? isPendingReimbursement : null
].filter(Boolean))
const healthExpenses = transactions.filter(filter)
```

Reply:
TIL `overEvery`, thanks